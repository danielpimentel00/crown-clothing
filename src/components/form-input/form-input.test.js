import { shallow } from "enzyme";
import FormInput from "./form-input.component";

describe("Form input component", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<FormInput />);

    expect(wrapper).toMatchSnapshot();
  });
});
