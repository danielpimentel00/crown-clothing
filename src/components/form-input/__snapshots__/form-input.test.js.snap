// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form input component should match snapshot 1`] = `ShallowWrapper {}`;
